src/main/kotlin/racingcar/domain/RaceGame.kt

```diff
@@ -0,0 +1,13 @@
+package racingcar.domain
+
+class RaceGame(val race: Race) {
+    val raceRecords = RaceRecords()
+    private val randomNumberGenerator = RandomNumberGenerator()
+
+    fun doRaceGame(numberOfTry: Int) {
+        repeat(numberOfTry) {
+            race.doRace(randomNumberGenerator.getRandomNumbers(race.cars.size))
+            raceRecords.raceRecords.add(RaceRecord(race = race))
```

Comment:
```suggestion
            raceRecords.add(RaceRecord(race))
```

raceRecords에서 raceRecords를 이용하여 RaceRecord를 더한다는 부분보다, raceRecords에 add 함수를 만들어서 raceRecords 프로퍼티를 감추어 캡슐화해 주는 건 어떨까요?